recipes/lzip/all/conanfile.py

```diff
@@ -1,102 +1,59 @@
-from conans import AutoToolsBuildEnvironment, CMake, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
-import contextlib
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.files import copy, get, replace_in_file, rmdir, patch, apply_conandata_patches, export_conandata_patches
+from conan.tools.layout import basic_layout
 import os
-import textwrap
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.53.0"
 
 
 class LzipConan(ConanFile):
     name = "lzip"
     description = "Lzip is a lossless data compressor with a user interface similar to the one of gzip or bzip2"
-    topics = ("lzip", "compressor", "lzma")
     license = "GPL-v2-or-later"
-    homepage = "https://www.nongnu.org/lzip/"
     url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.nongnu.org/lzip/"
+    topics = ("compressor", "lzma")
+    package_type = "application"
     settings = "os", "arch", "compiler", "build_type"
 
-    exports_sources = "patches/**"
-
-    _autotools = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _settings_build(self):
-        return getattr(self, "settings_build", self.settings)
+    def export_sources(self):
+        export_conandata_patches(self)
 
-    def build_requirements(self):
-        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
-            self.build_requires("msys2/cci.latest")
```

Comment:
Why is it removed (and not migrated to conan v2)? It can't build out of the box anymore on Windows if consumer doesn't have msys2